deps/boost/CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ if(WIN32)
   set(Boost_USE_STATIC_RUNTIME     OFF)
 endif()
 
+if(DEFINED ENV{DOCKER})
+  set(Boost_USE_STATIC_LIBS ON)
+endif()
```

Comment:
Can you please explain the reason for it?

Reply:
Statically compiling boost into the binaries rather than installing them in the runtime container saves a good amount of disk space